sirepo/auth/__init__.py

```diff
@@ -579,6 +579,9 @@ def _auth_hook_from_header(values):
 
 
 def _auth_state():
+    def get_slack_uri():
+        return sirepo.feature_config.cfg().sim_common.join_slack_uri \
+               + (logged_in_user() if _is_logged_in() else 'login')
```

Comment:
Let's just not append anything in the case of a guest. You should use `_get_user` for this:

```py
               + (_get_user() or '')
```

We want to track the user whether they are logged in or not.

Reply:
1e63e3e61d6b75f5c183715519625d2fdca71b42